enhancements/update/dont-require-registry-during-boot-and-upgrade.md

```diff
@@ -0,0 +1,201 @@
+---
+title: dont-require-registry-during-boot-and-upgrade
+authors:
+- "@jhernand"
+reviewers:
+- "@avishayt"
+- "@danielerez"
+- "@mrunalp"
+- "@nmagnezi"
+- "@oourfali"
```

Comment:
please indicate what aspects of the EP each of these reviewers should be focused on

Reply:
A YAML comment or something else?

Reply:
yaml comment, you can see an example here:
https://raw.githubusercontent.com/openshift/enhancements/master/enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```
reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
  - "@joelanford" # for OLM, OperatorHub changes
  - "@bparees" # for OLM, OperatorHub changes
  - "@csrwng" # for HyperShift changes
  - "@spadgett" # for Console changes
  - "@abutcher" # for CCO changes
  ```

Reply:
Done. Thanks for the example. I was looking at the rendered template so I missed this completely.